src/main/java/uk/gov/ons/ctp/response/sample/party/PartyUtil.java

```diff
@@ -32,8 +32,16 @@ public static PartyCreationRequestDTO convertToParty(SampleUnitBase unit) throws
       businessSampleUnit.setRusic92(bsu.getRusic92());
       businessSampleUnit.setFrosic2007(bsu.getFrosic2007());
       businessSampleUnit.setRusic2007(bsu.getRusic2007());
-      businessSampleUnit.setFroempment(Integer.valueOf(bsu.getFroempment()));
-      businessSampleUnit.setFrotover(Integer.valueOf(bsu.getFrotover()));
+      try {
+        businessSampleUnit.setFroempment(Integer.valueOf(bsu.getFroempment()));
+      } catch (NumberFormatException nfe) {
+        businessSampleUnit.setFroempment(0);
+      }
+      try {
+        businessSampleUnit.setFrotover(Integer.valueOf(bsu.getFrotover()));
+      } catch (NumberFormatException nfe) {
+        businessSampleUnit.setFrotover(0);
```

Comment:
Setting these to 0 by default may cause confusion down the line, can we just leave them as null?

Reply:
yeah that is probably better, I'll have to test though. I'm not 100% sure where these fields are used

Reply:
I don't think these fields are used anywhere currently just sit in the business attributes table.

Reply:
So `int`'s can't be set to null. I've seen it recommended to be set to 0 or -1. Not sure what's preferable here I'll try a few things out